Framework/PythonInterface/plugins/algorithms/IndexSatellitePeaks.py

```diff
@@ -0,0 +1,142 @@
+import numpy as np
+from scipy.spatial import KDTree
+import fractional_indexing as indexing
+
+from mantid.kernel import Direction
+from mantid.api import (IPeaksWorkspaceProperty,
+                        ITableWorkspaceProperty, PythonAlgorithm, AlgorithmFactory)
+import mantid.simpleapi as api
+
+
+class IndexSatellitePeaks(PythonAlgorithm):
+
+    def category(self):
+        return 'Crystal\\Peaks'
```

Comment:
Shouldn't this algorithm also override `name`, `summary`, `version` and perhaps `seeAlso`?